inst/private/python_ipc_driver.m

```diff
@@ -73,5 +73,10 @@
       [A, db] = python_ipc_sysoneline(what, cmd, false, varargin{:});
 
     otherwise
-      error('invalid ipc mechanism')
+      if (strcmp(what, 'reset'))
+        A = true;
+        db = [];
+      else
+        error('invalid ipc mechanism')
```

Comment:
Yeah, good catch. I copy-pasted the `strcmp` line from one of the other drivers, and just indented the else without looking at the style.